code/datums/looping_sounds/looping_sound.dm

```diff
@@ -0,0 +1,69 @@
+/datum/looping_sound
```

Comment:
huge overkill, just set ```repeat``` on a ```/sound```

Reply:
wait hold on, I asked fucking 4 times if looping sounds we a possible thing and no one said this?

Regardless, this doesn't work when the loop consists of random sounds

Reply:
well then that's a ```/datum/looping_RANDOM_sound``` then :^)
and of course nobody said, now I'm gone a huge chunk of knowledge has come with me :3

Reply:
it's not just random though, this also supports sounds that happen before and after the loop starts so you can have a transition.

Reply:
well that's just 3 sounds.
you've already validated why the datum exists, you can stop trying now.

Reply:
also making these kinds of sounds their own datums makes it easier to change machinery droning sounds etc across the whole code at once.